config/graylog.conf

```diff
@@ -190,7 +190,6 @@ http_bind_address = 0.0.0.0:9000
 #
 # Default: http://127.0.0.1:9200
 #elasticsearch_hosts = http://node1:9200,http://user:password@node2:19200
-elasticsearch_hosts = http://elasticsearch:9200
```

Comment:
@janheise Does this still work with regular OpenSearch setups in 5.2?

Reply:
as far as I tested everything, that still works. if we're not running in Preflight and we don't have that setting, we default to `"http://127.0.0.1:9200"` in `IndexerDiscoveryProvider`

Reply:
Okay, thanks! :+1: